script/mail_fetcher

```diff
@@ -51,9 +51,4 @@ rescue StandardError => e
   raise
 end
 
-if handler.errors.any?
-  Rails.logger.error(handler.errors.join("\n"))
-  NoisyWorkflow.report_errors(handler.errors).deliver
```

Comment:
Do we want to move these two lines to `lib/fact_check_email_handler.rb` after notifying Sentry?
Not sure what the impact is of removing them... would be a shame for NoisyWorkflow to be noisy in places and silent in others :)

Reply:
The `NoisyWorkflow` class really puzzles me. It's a mailer that sends notification emails to publisher, but also has this "make_noise" method, [added in 2011 for no documented reason](https://github.com/alphagov/publisher/commit/163b0dea17ecf66dfa2e15bc3349b0992b19f39f). Since then, we appear to have shoved all the other (useful) notifications into this class, such that its original name is inaccurate (and quite unhelpful, frankly).

Anyway, as long as we're capturing the errors (in Sentry), I don't see the value in sending ourselves an email about it - even if it did work.